lib/ads/interstitial_ad_manager.js

```diff
@@ -598,6 +609,27 @@ shaka.ads.InterstitialAdManager = class {
   }
 
 
+  /**
+   * @private
+   */
+  cuepointsChanged_() {
+    /** @type {!Set.<!shaka.extern.AdCuePoint>} */
+    const cuePoints = new Set();
+    for (const interstitial of Array.from(this.interstitials_)) {
+      /** @type {shaka.extern.AdCuePoint} */
+      const shakaCuePoint = {
+        start: interstitial.startTime,
+        end: null,
+      };
+      cuePoints.add(shakaCuePoint);
+    }
+
+    this.onEvent_(new shaka.util.FakeEvent(
+        shaka.ads.AdManager.CUEPOINTS_CHANGED,
+        (new Map()).set('cuepoints', Array.from(cuePoints))));
```

Comment:
Incidentally, I was looking up our documentation on the `ad-cue-points-changed` event, and there are some major typos in it:

1. It says it has a property called `originalEvent` that contains the original IMA event. That's wrong, the event should contain an event called `cuepoints` as you made it here.
2. It says that the qualified name of the event is `shaka.ads.AdManager.AdResumedEvent`. That appears to be a copy-paste error, it should probably be `shaka.ads.AdManager.AdCuePointsChangedEvent`.

Reply:
Changed!